config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,71 @@
+class Rack::Attack 
```

Comment:
I noticed this reopens the Rack::Attack class (because their example shows that), but I think it's cleaner to just call `Rack::Attack.throttle` directly (as in their README)

Reply:
Yeah, the example came from the ["Usage" section](https://github.com/rack/rack-attack/tree/6-stable#usage) in the README, but I don't have a strong opinion changing it to that model.

The only thing is that constants are being used currently to hold some of these values, but I will see if they are needed long term or if they can go away.

Reply:
Addressed.